agents/appmotion_motor/appmotion_motors_agent.py

```diff
@@ -0,0 +1,1142 @@
+            if params['motor'] == 1:
+                self.motor1.seek_home_linear_stage()
+            elif params['motor'] == 2:
+                self.motor2.seek_home_linear_stage()
+            elif params['motor'] == 3:
+                self.motor1.seek_home_linear_stage()
+                self.move_status = self.motor1.is_moving(params['verbose'])
+                while self.move_status:
+                    self.move_status = self.motor1.is_moving(params['verbose'])
+                self.motor2.seek_home_linear_stage()
+            else:
+                print("Motor ID invalid argument")
+
+        return True, "Moving motor {} to home".format(params['motor'])
+
+    @ocs_agent.param('motor', default=1, type=int)
+    @ocs_agent.param('verbose', default=True, type=bool)
+    def set_zero(self, session, params):
+        """set_zero(motor=1)
+
+        **Task** - Sets the zero position (AKA home) for motor(s) specified in
+        params.
+
+        Parameters:
+            motor (int): Determines which motor, either 1 or 2, 3 is for all
+                motors. (default 1)
+        """
+
+        self.move_status = self.is_moving(session, params)[1][1]
+        with self.lock.acquire_timeout(1, job=f"set_zero_motor{params['motor']}") as acquired:
+            if self.move_status:
+                return False, "Motors are already moving."
+            if not acquired:
+                self.log.warn(
+                    f'Could not set_zero because lock held by {self.lock.job}')
+                return False, 'Could not acquire lock'
+            if params['motor'] == 1:
+                self.motor1.set_zero()
+            elif params['motor'] == 2:
+                self.motor2.set_zero()
+            elif params['motor'] == 3:
+                self.motor1.set_zero()
+                self.move_status = self.motor1.is_moving(params['verbose'])
+                while self.move_status:
+                    self.move_status = self.motor1.is_moving(params['verbose'])
+                self.motor2.set_zero()
+            else:
+                print("Motor ID invalid argument")
+
+        return True, "Zeroing motor {} position".format(params['motor'])
+
+    @ocs_agent.param('motor', default=1, type=int)
+    @ocs_agent.param('pos_is_inches', default=True, type=bool)
+    @ocs_agent.param('pos_data', type=list)
+    @ocs_agent.param('verbose', default=True, type=bool)
+    def run_positions(self, session, params):
+        """run_positions(pos_data=None, motor=1, pos_is_inches=False)
+
+        **Task** - Runs a list of entries as positions. For
+        motor=ALL, the first column must be the x-data, and the second column
+        the y-data. Each position will be attained.
+
+        Parameters:
+            motor (int): Determines which motor, either 1 or 2, 3 is for all
+                motors. (default 1)
+            pos_data (list): Tab-delimited list of entries. First column
+                is x-data, second column is y-data.
```

Comment:
So how it's used in scripts:
Give a list of up to two elements. If 2 elements exist, then motor _must_ == 3, and motor1 moves to the first element, and motor2 moves to the second element. 

It's different than the other movement tasks because you can give 2 different positions to the motors. There is currently no capability to run multiple positions to one motor